cypress/e2e/tests/pages/user-menu/preferences.spec.ts

```diff
@@ -413,45 +360,43 @@ describe('User can update their preferences', () => {
     }
   });
 
-  describe('Check keyboard mapping on YAML Editor', { tags: ['@userMenu', '@adminUser', '@standardUser'] }, () => {
```

Comment:
I moved all of this to be in the middle and out of the describe block because adding the login/logout tests for the login landing page was causing problems with the sessioned login attempts and I this was the only way I thought of resolving it without disabling all of the sessioning for these tests.